### DISCUSSION TITLE
Add Charging Sessions

### DISCUSSION THREAD

**Discussion by sincze** [bimmerconnected/bimmer_connected#633] [Feature request]
I used to obtain my historic charging session information via a MyBMW Node-Red integration. However this stopped working.

<img width="853" alt="image" src="https://github.com/user-attachments/assets/03c537a9-e9a7-4e70-8fb6-7ddb7d8aa3ce">

Am I missing something here that it could also be supported in HA or CLI so I can process the results and update my persistent counters ;-) ?

**Comment by rikroe** [comment]
Currently not implemented. There was a [PR](https://github.com/bimmerconnected/bimmer_connected/pull/569) that got closed.
Can have a look but no promises regarding time.
